src/shared/components/AssetsManagementList/AssetsManagementListItem.tsx

```diff
@@ -0,0 +1,54 @@
+
+import React, { useEffect, useState } from 'react'
+import { Storage } from 'aws-amplify'
+
+import awsmobile from '../../../aws-exports'
+import { fetchVodSections } from '../../utilities'
+import Loader from 'react-loader-spinner'
+
+const AssetsManagementListItem = ({ selectedAsset }: any) => {
+    const [thumbnail, setThumbnail] = useState(
+        <Loader type="Rings" color="#FFA41C" height={100} width={100} timeout={3000} />
+    )
+    const [sections, setSections] = useState<Array<any>>([])
+
+    useEffect(() => {
+        const thumb: any = Storage.get(`thumbnails/${selectedAsset.thumbnail.id}.${selectedAsset.thumbnail.ext}`, {
```

Comment:
Please can we use async/await with promises ? 
Also can you type thumb, instead of `any`.

Reply:
I noticed that thumb is not used